pwem/protocols/protocol_import/volumes.py

```diff
@@ -468,19 +468,29 @@ def createOutputStep(self, atomStructPaths):
       """ Copy the PDB structures and register the output object.
       """
       outASs = emobj.SetOfAtomStructs().create(self._getPath())
+      
+      numStructs = len(atomStructPaths)
+      padding_format_str = '%0' + str(len(str(numStructs))) + 'd'
+
       for atomStructPath in atomStructPaths:
-          if not exists(atomStructPath):
-              raise Exception("Atomic structure not found at *%s*" % atomStructPath)
+        if not exists(atomStructPath):
```

Comment:
Indentation does not seem correct?

Reply:
I was having a lot of problem with the indentation in this file. I'll have another look. Thanks

Reply:
It should be right now